src/core/dom/query.js

```diff
@@ -331,3 +331,64 @@ export function elementByTag(element, tagName) {
   assertIsName(tagName);
   return element./*OK*/ querySelector(tagName);
 }
+
+/**
+ * Returns the original nodes of the custom element without any service
+ * nodes that could have been added for markup. These nodes can include
+ * Text, Comment and other child nodes.
+ *
+ * @param {!Node} element
+ * @return {!Array<!Node>}
+ */
+export function getRealChildNodes(element) {
+  return childNodes(element, (node) => !isInternalOrServiceNode(node));
+}
+
+/**
+ * Returns the original children of the custom element without any service
+ * nodes that could have been added for markup.
+ *
+ * @param {!Element} element
+ * @return {!Array<!Element>}
+ * @package @final
```

Comment:
Do these `@package @final` have meaning now that they are no longer class members?

Reply:
Good callout! `@final` doesn't but `@package` still does. It doesn't make sense though, so I'll remove them.